completions/ip

```diff
@@ -8,11 +8,21 @@ _comp_cmd_ip__iproute2_etc()
 
 _comp_cmd_ip__netns()
 {
-    _comp_compgen_split -- "$(
+    local unquoted
+    _comp_split -l unquoted "$(
         {
             ${1-ip} -c=never netns list 2>/dev/null || ${1-ip} netns list
-        } | _comp_awk '{print $1}'
+        } | command sed -e 's/ (.*//g'
     )"
+    # namespace names can have spaces, so we quote all of them if needed
+    local i quoted=()
+    for i in "${!unquoted[@]}"; do
```

Comment:
Should we directly iterate over the values instead of the keys (because we do not use the key value)?

Reply:
Makes sense, changed it